concepts/bools/about.md

```diff
@@ -1,21 +1,25 @@
 # About
 
-Python represents True and False values with the [bool][bool] type. There are only two boolean values: `True` and `False`. These values can be assigned to a variable and combined with the [boolean operators][boolean-operators] (`and`, `or`, `not`):
+Python represents True and False values with the [bool][bool] type.
+ There are only two boolean values in this type: `True` and `False`.
+  These values can be assigned to a variable and combined with the [boolean operators][boolean-operators] (`and`, `or`, `not`):
```

Comment:
I am not positive, but I believe "Boolean" is meant to be capitalized, due to honoring the name of the inventor.  However, I am pretty sure I am not mistaken.

Reply:
you are correct!  Let me get that changed...

Reply:
```suggestion
  These values can be assigned to a variable and combined with the [Boolean operators][boolean-operators] (`and`, `or`, `not`):
```